android-lib/src/androidTest/assets/kinvey.properties

```diff
@@ -1,6 +1,6 @@
 # Required settings
-app.key=<YOUR_APP_KEY_HERE>
-app.secret=<YOUR_APP_SECRET_HERE>
+app.key=kid_BkssrV2eW
```

Comment:
These keys are needed, they're used in tests. Sure, moving it to Travis environment variables.